files/es/games/techniques/index.md

```diff
@@ -2,7 +2,7 @@
 title: Técnicas para el desarrollo de juegos web
 slug: Games/Techniques
 l10n:
-  sourceCommit: '048f6b1c75e22103ddb0304d67ee79d6d8a014f0'
+  sourceCommit: "048f6b1c75e22103ddb0304d67ee79d6d8a014f0"
```

Comment:
Not sure if this value has to be with or without
But we prefer without

```suggestion
  sourceCommit: 048f6b1c75e22103ddb0304d67ee79d6d8a014f0
```

Reply:
I think that this should be fine without quotes!